ohsome/clients.py

```diff
@@ -317,6 +329,15 @@ def _handle_request(self) -> OhsomeResponse:
             )
 
         except requests.exceptions.RequestException as e:
+            if isinstance(e, RetryError) and not self.__recursion_breaker:
+                # retry one last time without retries, this will raise the original error instead of a cryptic retry
```

Comment:
This is counterintuitive about the custom `Retry` config. Library users could be surprised by a subsequent call, e.g. when testing their functionality. Why do we need to retry once again?

Reply:
`urllib3` shadows the retry-causing exception. We target novice users with the package and they can't cope with `MaxRetriesExcpetion: too many 500 errors` . Also we have a sophisticated setup to catch errors and add meaningful information to it (maybe over-engineered ;-) ).

I will document the behaviour. The alternative would be to reduce the total number of retries given by users by 1.